pkgs/build-support/php/build-composer-project.nix

```diff
@@ -62,7 +62,7 @@ let
         runHook postInstallCheck
       '';
 
-      composerRepository = phpDrv.mkComposerRepository {
+      composerRepository = phpDrv.mkComposerRepository ({
```

Comment:
I would use `previousAttrs.composerRepository or phpDrv.mkComposerRepository …`. This looks more like `composerRepositoryArgs` and having different behaviour when passing it to `buildComposerProject` and to `overrideAttrs` would be confusing.